development/templates/development/match_details.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% block page_content %}    
+<div class="container">
+    <h1>Match details</h1>
+    <div class="pre-scrollable">
+    <table 
+        class="table table-striped table-bordered table-sm"
+        style="width: 100%">
+        <thead>
+        <tr class="bg-primary text-white" style="
+            position: sticky;
+            top: 0;">
+            <th scope="col">State</th>
+            <th scope="col">Turn</th>
+            <th scope="col">Player1</th>
+            <th scope="col">Score1</th>
+            <th scope="col">Player2</th>
+            <th scope="col">Score2</th>
+            <th scope="col">Origin</th>
+            <th scope="col">Destination</th>
+            <th scope="col" style="width: 15%;">Board</th>
+          </tr>
+        </thead>
+        <tbody>
+        {% for log in data %}
+            <tr>
+                <td>{{ log.state }}</td>
+                <td>{{ log.current_player }}</td>
+                <td>{{ log.player_1 }}</td>
+                <td>{{ log.score_1 }}</td>
+                <td>{{ log.player_2 }}</td>
+                <td>{{ log.score_2 }}</td>
+                <td>({{ log.from_row }}, {{ log.from_col }})</td>
+                <td>({{ log.to_row }}, {{ log.to_col }})</td>
+                <td style="font-family:'sans-serif', monospace">{{ log.board }}</td>
```

Comment:
nit: ideally, we should apply styles in a CSS class.

Reply:
done!